db_scripts/sql/01_eventsourcing.sql

```diff
@@ -29,7 +29,12 @@ CREATE TABLE IF NOT EXISTS events
     -- Event name, e.g. BusCreated, HotelReservationCountChanged
     event_name VARCHAR NOT NULL,
     -- Event data blob - JSON object for *Created events, (JSON) integer for *Changed events
-    data JSONB NOT NULL,
+    data VARCHAR NOT NULL,
 
     change_time TIMESTAMP NOT NULL DEFAULT now()
 );
+
+CREATE SEQUENCE public.seq
+INCREMENT 1
+START 700000
+MINVALUE 1;
```

Comment:
I was wondering about the reason this was created for... Is it done to avoid the DB filling script from failing if we try adding a record while it's still running?

Reply:
Nope, currently scripts use hardcoded id value, so in order for auto increment to work I had to initialize sequence somewhere after last inserted key